tx/textual/valuerenderer/types.go

```diff
@@ -34,3 +34,15 @@ type ValueRenderer interface {
 	// Parse should be the inverse of Format.
 	Parse(context.Context, []Screen) (protoreflect.Value, error)
 }
+
+// RepeatedValueRenderer defines an interface to produce formatted output for
+// protobuf message fields that are repeated.
+type RepeatedValueRenderer interface {
+	ValueRenderer
+
+	// FormatRepeated should render the value to a text plus annotation.
+	FormatRepeated(context.Context, protoreflect.Value) ([]Screen, error)
+
+	// ParseRepeated should be the inverse of Format.
+	ParseRepeated(context.Context, []Screen, protoreflect.List) (protoreflect.Value, error)
```

Comment:
```suggestion
	ParseRepeated(context.Context, []Screen) (protoreflect.Value, error)
```

I think this signature is enough, no need to have 2 times the list value.

The alternative would be:
```
ParseRepeated(context.Context, []Screen, protoreflect.List) error
```

but I think I prefer the former

Reply:
I wasn't sure how the implementer `ParseRepeated` was going to create a List, so I opted to pass it in with the expectation that it will be populated so that the field may be set:

https://github.com/cosmos/cosmos-sdk/pull/13604/files#diff-7c38c5aa4e5372b066118d3d77aa34814c151f9d5c591fadd42f64c7687849b6R230-R242

Reply:
point taken though.. lemme try to reduce the signature